kernel/src/analysis/javascript.rs

```diff
@@ -215,12 +216,6 @@ pub fn execute_rule(
 
     yield_now();
```

Comment:
Is this really needed?

Reply:
Also, not related to this line but to line 243 (cannot put a comment there), why the explicit `drop`? The `JoinHandle` is going to be dropped anyway.

Reply:
This was something that was related to the previous code. Removing it, thank you for flagging it :)